frontend/packages/ux-editor/src/containers/FormDesigner.tsx

```diff
@@ -105,20 +111,30 @@ export const FormDesigner = ({
 
   if (formLayoutIsReady) {
     const triggerDepthAlert = () => alert(t('schema_editor.depth_error'));
+    const triggerInvalidChildAlert = () => alert(t('schema_editor.invalid_child_error'));
     const layout = formLayouts[selectedLayout];
 
     const addItem: HandleAdd<ComponentType> = (type, { parentId, index }) => {
       const newId = generateComponentId(type, formLayouts);
 
       const updatedLayout = addItemOfType(layout, type, newId, parentId, index);
-      if (validateDepth(updatedLayout)) {
+      if (validateDepth(updatedLayout) && validateContainerChild(updatedLayout, parentId, type)) {
         addItemToLayout({ componentType: type, newId, parentId, index });
         handleEdit(getItem(updatedLayout, newId));
-      } else triggerDepthAlert();
+      } else {
+        if (!validateDepth(updatedLayout)) triggerDepthAlert();
+        if (!validateContainerChild(updatedLayout, parentId, type)) triggerInvalidChildAlert();
+      }
     };
     const moveItem: HandleMove = (id, { parentId, index }) => {
       const updatedLayout = moveLayoutItem(layout, id, parentId, index);
-      validateDepth(updatedLayout) ? updateFormLayout(updatedLayout) : triggerDepthAlert();
+      const type = getItem(layout, id).type;
```

Comment:
Is it possible to add tests for the lines mentioned by Codecov in this file? Line ```131```, and ```133```